src/amuse/support/__init__.py

```diff
@@ -26,3 +26,11 @@ def get_amuse_root_dir():
 
 def get_amuse_data_dir():
     return _Defaults().amuse_root_dir
+
+def get_amuse_directory_root():
```

Comment:
easily confused with get_amuse_root_dir, not an ideal name. I get it existed before, but especially when they're in the same module I would prefer clearer names (or at least a docstring that explains the difference)...